relay/messages/execution.go

```diff
@@ -17,7 +17,6 @@ type ExecutionRequest struct {
 	InvocationID   string                 `json:"invocation_id"`
 	InvocationStep string                 `json:"invocation_step"`
 	Command        string                 `json:"command"`
-	CommandConfig  map[string]interface{} `json:"command_config"`
```

Comment:
This was an unused config var yes. I'm guessing it was meant to include env vars for commands, but we already have those in the config anyway so we don't need it with the execution request.